docker-compose.yml

```
@@ -0,0 +1,112 @@
+version: "3.9"
+services:
+
+  #
+  # Portainer:
+  # Used for managing Docker containers and administration.
+  #
+  portainer:
+    container_name: portainer
+    image: portainer/portainer:latest
+    networks:
+      - darkflame_net
+    ports:
+      - "9000:9000"
+    volumes:
+      - portainer_data:/data
+      - /var/run/docker.sock:/var/run/docker.sock
+
+  #
+  # MariaDB:
+  # A port of MySQL used as the database
+  #
+  mariadb:
+    command: --default-authentication-plugin=mysql_native_password
+    container_name: mariadb
+    image: mariadb:latest
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+    networks:
+      - darkflame_net
+    ports:
+      - "3306:3306"
+    restart: always
+    volumes:
+      - db_data:/var/lib/mysql
+      - db_conf:/etc/mysql
```

I think it could be a good idea to include a default database and user.
So there is no need to create one for the people that dont know how.

```suggestion
  mariadb:
    command: --default-authentication-plugin=mysql_native_password
    container_name: mariadb
    image: mariadb:latest
    environment:
      - MYSQL_ROOT_PASSWORD: password
      - MYSQL_DATABASE: darkflame
      - MYSQL_USER: darkflame
      - MYSQL_PASSWORD: darkflame_pw
    networks:
      - darkflame_net
    ports:
      - "3306:3306"
    restart: always
    volumes:
      - db_data:/var/lib/mysql
      - db_conf:/etc/mysql
```

---

That's actually a great idea! Also, I only included Portainer for debugging — most won't care, but it might make it easy if we need someone to SSH into a container, for instance.

Although we can always remove if if you feel it's still unnecessary.